java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/SSL/EndpointFactoryI.java

```diff
@@ -9,6 +9,9 @@
 
 import java.util.ArrayList;
 
+/**
+ * @hidden Used by Ice
```

Comment:
Used by Ice is a bit too generic. Presumably, it's public and hidden because it's just by some code outside the Ice package.

Reply:
In this case it's used by Ice as it's in the IceSSL package.